src/Symfony/Component/AssetMapper/ImportMap/Providers/JsDelivrEsmImportMapProvider.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\ImportMap\Providers;
+
+use Symfony\Component\AssetMapper\ImportMap\PackageRequireOptions;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @experimental
```

Comment:
```suggestion
 * @experimental
 * @final
```

Reply:
We could even use an actual `final` keyword (any mocking can be done on the interface anyway)